data/package_index.yml

```
@@ -1626,3 +1626,11 @@
   tags: quasi-newton-optimization stochastic-optimization acm-toms
   license: MIT
   version: none
+
+- name: FLAME
+  gitlab: flame-code/FLAME
+  description: FLAME: a library for atomistic modeling environments.
+  categories: atomistic simulation
+  tags: Machine learning interatomic potential, Crystal structure prediction
```

```suggestion
  tags: machine-learning interatomic-potential crystal-structure-prediction
```
Tags should be separated by spaces.

Looking at other projects, multiword tags use hyphens.